test/relations.test.js

```diff
@@ -34,9 +34,16 @@ describe('relations', function() {
     db = getSchema();
   });
 
+  after(function(done) {
+     // disconnect from this db to avoid too many connection error
+     // due to multiple instance of connection pool
+    db.disconnect();
+    done();
+  });
+
   describe('hasMany', function() {
     before(function(done) {
-      db = getSchema();
+      // db = getSchema();
```

Comment:
just pushed, let's see.
@raymondfeng : what does this mentions stands for in the test files headers? (with regard to your statement on singleton)

`/* global getSchema:false, connectorCapabilities:false */`